### DISCUSSION TITLE
The error of streaming audio from my microphone

### DISCUSSION THREAD

**Discussion by iceleaf97tech** [deepgram/community#134] [General help] [answered]
I am sure that I have installed portaudio and pyaudio, but I still receive an error message when running this command:
`python test_suite.py -k YOUR_DEEPGRAM_API_KEY -i mic`

[ERROR Message]
ℹ️  Request ID: 7a563f3b-336c-4b8d-8183-00121531de95
🟢 (1/5) Successfully opened Deepgram streaming connection
🟢 (2/5) Ready to stream mic audio to Deepgram. Speak into your microphone to transcribe.
Error while sending: Task <Task pending name='Task-5' coro=<run.<locals>.sender() running at C:\Storage\Research\Deepgram\streaming-test-suite\test_suite.py:96> cb=[gather.<locals>._done_callback() at C:\Users\admin\AppData\Local\Programs\Python\Python39\lib\asyncio\tasks.py:761]> got Future <Future pending> attached to a different loop
🔴 ERROR: Something went wrong! Task <Task pending name='Task-5' coro=<run.<locals>.sender() running at C:\Storage\Research\Deepgram\streaming-test-suite\test_suite.py:96> cb=[gather.<locals>._done_callback() at C:\Users\admin\AppData\Local\Programs\Python\Python39\lib\asyncio\tasks.py:761]> got Future <Future pending> attached to a different loop

I tried downloading the latest version from the PortAudio website：v19.7.0 stable release
I compiled PortAudio using Visual Studio 2019 and obtained a portable_x64.dll file, located at the following file path:
(file path)\portaudio\build\msvc\x64\Release

[Tools]
pyaudio==0.2.13
portaudio v19.7.0
python==3.9.1

**Comment by shirgoldbird** [comment, marked as answer]
Hi @iceleaf97tech! Unfortunately, we've run into struggles getting portaudio to run on Windows environments, even when it seems to be installed correctly. You may want to try running it in WSL, or skipping the microphone streaming step in the streaming test suite. Portaudio is only required for that portion of the test suite, and everything else should work fine.

If you have any other issues, or general feedback about the streaming test suite, I'd love to hear them. Let me know if we can help you with anything else.

**Reply by iceleaf97tech** [reply]
@shirgoldbird Thank you for your reply! I will try to follow your suggestion.
